_includes/devdoc/example_transactions.md

```diff
@@ -840,7 +849,9 @@ broadcasts it.
 > bitcoin-cli -regtest sendrawtransaction $SIGNED_RAW_TX
 {% endhighlight %}
 {% highlight json %}
-error: {"code":-22,"message":"TX rejected"}
+error code: -25
+error message:
+Missing inputs
 {% endhighlight %}
```

Comment:
Is this the original formatting of the error message? Mine is structured like this 

> 258: txn-mempool-conflict (code -26)